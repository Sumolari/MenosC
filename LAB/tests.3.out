Linea 1: syntax error, unexpected INT_, expecting BROP_
