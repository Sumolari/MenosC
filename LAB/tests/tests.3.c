int numero 7;
numero = -20;
numero++
numero = +20
numero--
numero = 5 / 7
numero = 5 * 7
numero = 5 + 7
numero = 5 - 7
