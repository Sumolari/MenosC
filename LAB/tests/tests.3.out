Linea 1: syntax error, unexpected INT_, expecting BROP_

Numero de errores:      1
